client/my-sites/store/components/woocommerce-colophon/index.js

```diff
@@ -1,30 +0,0 @@
-import { ExternalLink } from '@automattic/components';
-import { localize } from 'i18n-calypso';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import { recordTracksEvent } from 'calypso/state/analytics/actions';
-import WooCommerceLogo from '../woocommerce-logo';
-
-class WooCommerceColophon extends Component {
```

Comment:
Funny that this one was duplicated, there's another instance in the root of the `my-sites/woocommerce` dir.